### PR TITLE
Test Impact Analysis Framework clang build fixes

### PR DESCRIPTION
**Pull request by lumberyard-employee-dm**
## How was this PR tested?

Built TIAF using Windows clang 15.0.0

### CODE REVIEW

File: Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/TestImpactRepoPath.cpp
Comment (hultonha) [inline Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/TestImpactRepoPath.cpp:13]: uber nit: Looks like the indentation is slightly off here

File: Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestEngine/Native/TestImpactNativeTestEngine.cpp
Comment (hultonha) [inline Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestEngine/Native/TestImpactNativeTestEngine.cpp:68]: nit: Could we remove this new line break?

**Review by hultonha — APPROVED** [review]
Thank you for fixing these issues @lumberyard-employee-dm. I know @jonawals is out until next week so might not be able to provide feedback before then so if someone else can approve that would be great if this is blocking (I can poke some others to have a look at the PR). If anything needs to be changed I'm sure John can follow-up next week. Thanks!

**Review by cgalvan — APPROVED** [review]

File: Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/TestImpactRepoPath.cpp
Comment (lumberyard-employee-dm) [reply Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/TestImpactRepoPath.cpp:13]: Will fix

File: Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestEngine/Native/TestImpactNativeTestEngine.cpp
Comment (lumberyard-employee-dm) [reply Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestEngine/Native/TestImpactNativeTestEngine.cpp:68]: Yeah I'll remove this.

**Pull request merged**
